CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Add the concept of "propagatable slots".
 - Add the `curie_map` to the model (instead of it being a specificity of the SSSOM/TSV format).
 - Add the concept of "extension slots".
+- Add the concept of "literal mappings".
```

Comment:
- Replace the previous idea of a "literal profile" that is separate from the main SSSOM standard with a simple flag that allows publishing literal mappings in the standard SSSOM format

Reply:
Since the “previous idea of a ‘literal profile’” had never been officially announced in the changelog, and that profile had never been usable anyway (having never been connected to the rest of the spec), I am not sure we need to mention it. No objection, though.

Reply:
Ok! Your judgment. I am fine either way.